inputs/TransactionForm.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { Text, View, TextInput, TouchableOpacity, Platform } from 'react-native';
+import DateTimePicker from '@react-native-community/datetimepicker';
+import styles from '../styles';
+
+export default function TransactionForm() {
+    const [description, setDescription] = useState('');
```

Comment:
This is adding extra logic separately when we could combine everything in one object state, as we did in the inputs lesson, to maintain readability. How to set it as one state and then update was in the inputs lesson